### PR TITLE
Create southxchange.js

### PR DESCRIPTION
**Pull request by basti050295**

### CODE REVIEW

**Comment by joeuhren** [comment]
I had a chance to review this code today and I notice that it has numerous problems and is not formatted to work with the eIquidus explorer. It looks more like an exchange integration for the original iquidus explorer which is different enough that this will need to be fixed or rewritten. Not only that, but the api seems to target api v1 and should be updated to v3. There also seems to be a [history api endpoint](https://main.southxchange.com/Content/swagger/ui/?urls.primaryName=API%20v3#/Public%20data/get_history__listingCurrencyCode___referenceCurrencyCode___start___end___periods_) which should be used to populate the chart data. You can look at the existing eiquidus markets for some examples of how to implement the chart.

Long story short, please consider rewriting this if you would like this to be merged as it is broken and unusable in its current format.

**Comment by joeuhren** [comment]
Closing this request as southexchange support was officially added today: [https://github.com/team-exor/eiquidus/commit/56d4979383e89c3296a35cd36930b880bfd0c789](https://github.com/team-exor/eiquidus/commit/56d4979383e89c3296a35cd36930b880bfd0c789)
